src/main/kotlin/creator/custom/finalizers/RunGradleTasksFinalizer.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Minecraft Development for IntelliJ
+ *
+ * https://mcdev.io/
+ *
+ * Copyright (C) 2024 minecraft-dev
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, version 3.0 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.demonwav.mcdev.creator.custom.finalizers
+
+import com.demonwav.mcdev.util.runGradleTaskAndWait
+import com.intellij.openapi.diagnostic.thisLogger
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.project.guessProjectDir
+
+class RunGradleTasksFinalizer : CreatorFinalizer {
+
+    override fun execute(project: Project, properties: Map<String, Any>, templateProperties: Map<String, Any?>) {
+        @Suppress("UNCHECKED_CAST")
+        val tasks = properties["tasks"] as? List<String>
+            ?: return
```

Comment:
Validation could happen during the template setup, and if anything wrong happens during the execution either 
- a notification for template authors in case of a problem that has not been detected during the initial validation (if it ever happens)
- a warn/error, whichever makes sense if this is an issue with the implementation

Validation wasn't my concern early on, and this PR isn't done yet, it's functional but not fully ready (maybe I should make it a draft 🤔)